index.js

```diff
@@ -10,8 +10,8 @@ selector and the style property, write pure JavaScript to reveal
 who's behind the remaining boxes. 
 ________________________________________________________________*/
 
-// Your code here
-
+let boxTwo = (document.querySelector("#box-two").style.opacity = 1);
```

Comment:
no need to assign these to variables, but if you did, make sure you don't combine it with the style declaration. so either:
`document.querySelector("#box-two").style.opacity = 1`
or
`let boxTwo = document.querySelector("#box-two");`
`boxTwo.style.opacity = 1;`